TX.h

```diff
@@ -538,15 +538,18 @@ static inline void processSBUS(uint8_t c)
     if ((frameIndex == 23) && (c == SBUS_TAIL)) {
       uint8_t set;
       for (set = 0; set < 2; set++) {
-        PPM[(set<<3)] = ppmWork.sbus.ch[set].ch0 >> 1;
-        PPM[(set<<3)+1] = ppmWork.sbus.ch[set].ch1 >> 1;
-        PPM[(set<<3)+2] = ppmWork.sbus.ch[set].ch2 >> 1;
-        PPM[(set<<3)+3] = ppmWork.sbus.ch[set].ch3 >> 1;
-        PPM[(set<<3)+4] = ppmWork.sbus.ch[set].ch4 >> 1;
-        PPM[(set<<3)+5] = ppmWork.sbus.ch[set].ch5 >> 1;
-        PPM[(set<<3)+6] = ppmWork.sbus.ch[set].ch6 >> 1;
-        PPM[(set<<3)+7] = ppmWork.sbus.ch[set].ch7 >> 1;
+        PPM[(set<<3)] = (((ppmWork.sbus.ch[set].ch0 >> 1) * 5) >> 2);
+        PPM[(set<<3)+1] = (((ppmWork.sbus.ch[set].ch1 >> 1) * 5) >> 2);
+        PPM[(set<<3)+2] = (((ppmWork.sbus.ch[set].ch2 >> 1) * 5) >> 2);
+        PPM[(set<<3)+3] = (((ppmWork.sbus.ch[set].ch3 >> 1) * 5) >> 2);
+        PPM[(set<<3)+4] = (((ppmWork.sbus.ch[set].ch4 >> 1) * 5) >> 2);
+        PPM[(set<<3)+5] = (((ppmWork.sbus.ch[set].ch5 >> 1) * 5) >> 2);
+        PPM[(set<<3)+6] = (((ppmWork.sbus.ch[set].ch6 >> 1) * 5) >> 2);
+        PPM[(set<<3)+7] = (((ppmWork.sbus.ch[set].ch7 >> 1) * 5) >> 2);
       }
+      for(uint8_t i=0; i<16; i++){
+       PPM[i]=(PPM[i]<=108)?0:(PPM[i]-108);
```

Comment:
@mikeller,
fixed... I figured out what you meant. It turned out to be simpler & more correct to just use 'servoBits2Us()' function to handle the scaling logic. Tested over full range & it works consistently now. feedback would be appreciated.

Now, only other issue I'm thinking might exist is that perhaps the scaling should also be fixed on the RX side for SBUS output? That would mean editing the SBUS routine in 'SerialPPM.h'. Wondering if perhaps the SBUS-PPM encoding&decoding stuff should all go there?